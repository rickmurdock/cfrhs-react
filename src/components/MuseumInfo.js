import React from 'react';
import StationResized from '../images/StationResized.png';

const MuseumInfo = () => {
  return (
    <div>
      <div className="card" id="left">
        <img className="card-img-top img-station" src={StationResized} alt="Central Florida Railroad Museum Depot" />
        <div className="card-block">
          <h5>Museum Hours:</h5>
          <p>Daily from 1:00 PM - 5:00 PM<br/><sup>(Closed Select Holidays)</sup></p>
          <h5>Admission:</h5>
          <p>Free</p>
          <h5>Location:</h5>
          <p>101 S. Boyd Street<br/>Winter Garden, FL 34787</p>
          <p><i className="fa fa-phone" aria-hidden="true"></i> (407) 656-0559</p>
        </div>
      </div>
      <div>
        <iframe className="img-map" title="Map to museum" src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d14016.848628218677!2d-81.5829144!3d28.5633913!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xdf760d33e403cdfc!2sCentral+Florida+Railroad+Museum!5e0!3m2!1sen!2sus!4v1504388469093" allowFullScreen></iframe>
      </div>
    </div>
  );
}

export default MuseumInfo;