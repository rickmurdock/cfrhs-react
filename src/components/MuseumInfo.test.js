import React from 'react';
import renderer from 'react-test-renderer';
import MuseumInfo from './MuseumInfo';

test('MuseumInfo renders correctly', () => {
    const tree = renderer.create(
        <MuseumInfo />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});