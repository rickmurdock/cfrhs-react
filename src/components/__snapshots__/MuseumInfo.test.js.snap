// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MuseumInfo renders correctly 1`] = `
<div>
  <div
    className="card"
    id="left"
  >
    <img
      alt="Central Florida Railroad Museum Depot"
      className="card-img-top img-station"
      src="StationResized.png"
    />
    <div
      className="card-block"
    >
      <h5>
        Museum Hours:
      </h5>
      <p>
        Daily from 1:00 PM - 5:00 PM
        <br />
        <sup>
          (Closed Select Holidays)
        </sup>
      </p>
      <h5>
        Admission:
      </h5>
      <p>
        Free
      </p>
      <h5>
        Location:
      </h5>
      <p>
        101 S. Boyd Street
        <br />
        Winter Garden, FL 34787
      </p>
      <p>
        <i
          aria-hidden="true"
          className="fa fa-phone"
        />
         (407) 656-0559
      </p>
    </div>
  </div>
  <div>
    <iframe
      allowFullScreen={true}
      className="img-map"
      src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d14016.848628218677!2d-81.5829144!3d28.5633913!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xdf760d33e403cdfc!2sCentral+Florida+Railroad+Museum!5e0!3m2!1sen!2sus!4v1504388469093"
      title="Map to museum"
    />
  </div>
</div>
`;
